database.php

```diff
@@ -0,0 +1,9 @@
+<?php
+$db = new PDO('mysql:host=db;dbname=guitarApp','root', 'password');
```

Comment:
Have a think about whether or not you could create some functions for this code? 

What about if you had one function that creates a database connection, and then another function that loads the list of guitars? At the moment there would be no way to get a database connection without also getting a list of guitars